README.md

```diff
@@ -55,6 +60,7 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 ## Authors
 
 * [Harold Martin](https://www.linkedin.com/in/harold-martin-98526971/) - harold.martin at gmail
+* Jonah Caplan
```

Comment:
@jcaplan lmk if you're ok with this or want to include some other info

Reply:
Haha I'm flattered! I'm ok with just a name. Prefer to keep my email private.